matlab/README.md

```diff
@@ -0,0 +1,9 @@
+# {{ name }}
+
+{% if __project_description__ %}
+{{ __project_description__ }}
+{% endif %}
```

Comment:
this is missing from the desktop readme above - probably want them to be consistent?